cypress/e2e/components/movieListPage.cy.ts

```diff
@@ -0,0 +1,68 @@
+context('Movie List Page component', () => {
+  describe('rendering', () => {
+    beforeEach(function () {
+      cy.visit('/');
+    });
+
+    const movie = 'Pandas';
+
+    it('should render search initially', () => {
+      cy.contains('search');
+      cy.get('input[placeholder*="What do you want to watch?"]');
+    });
+
+    it('shoud contain expected movie after search', () => {
+      cy.get('[data-testid="search-input"]').type(movie);
+
+      cy.get('[data-testid="search-input"]').type('{enter}');
+      cy.contains(movie);
+    });
+
+    it('shoud contain ¡Three Amigos! after sort by title selected', () => {
+      cy.get('[data-testid="dropdown"]').click();
+
+      cy.get('div').contains('Title').click();
+
+      cy.get('[data-testid="search-input"]').type('{enter}');
+      cy.contains('¡Three Amigos!');
+    });
+
+    it('shoud contain The Gold Rush 1925 yesr release after sort by release date selected', () => {
+      cy.get('[data-testid="dropdown"]').click();
+
+      cy.get('div').contains('Title').click();
+      cy.get('[data-testid="dropdown"]').click();
+      cy.get('div').contains('Release Date').click();
+
+      cy.contains('The Gold Rush');
+      cy.contains('1925');
+    });
+
+    it('shoud contain all 10 movies with if filtere by genre', () => {
+      cy.get('button').contains('Documentary').click();
+
+      cy.get('[data-testid="moviegenres"]').should('contain', 'Documentary');
```

Comment:
You do not check films list here as described in title.

Reply:
Fixed titile, what I want to test here, is that if we filter by genre, then this genre will be in each movie genres list.